scania-digital-design-system/tegel
# feat(chip): disabled state for chip

## **Describe pull-request**  
Adding disabled state for chip aligning with already existing design. 
## **Issue Linking:**  
- **Jira:** [CDEP-3238](https://tegel.atlassian.net/browse/CDEP-3238)    

## **How to test**  
1. Go to Storybook
2. Check in Chip
3. Toggle disabled boolean and see that it applies correctly for all types and aligns with Figma design.


[CDEP-3238]: https://tegel.atlassian.net/browse/CDEP-3238?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
